### DISCUSSION TITLE
helm-mini -- search by buffer mode

### DISCUSSION THREAD

**Discussion by pedz** [emacs-helm/helm#2502] [Q&A] [answered]
I start helm-mini via `C-x b` and the bottom line has `pattern` and I can type and it searches in the buffer name field.  Fine.

Two columns over (3rd column) I guess is the mode the buffer is in.  Can I search on that field.  This is a specific example of a more generic question of how to search in other fields -- although, helm-mini might be the only one that has multiple fields to search in.

The other question is about Github... how can I get notifications of new discussions here?

Oh... sorry, one last question: I date back to when memory was precious so killing buffers is a thing I do.  Call it a mental illness.  Can I kill any buffer with a name starting with `*helm-`.  I think I read somewhere that I could but just wanted to check.

**Comment by thierryvolpiatto** [comment, marked as answer]
Perry Smith ***@***.***> writes:

> I start helm-mini via C-x b and the bottom line has pattern and I can
> type and it searches in the buffer name field. Fine.
>
> Two columns over (3rd column) I guess is the mode the buffer is
> in. Can I search on that field.

Yes, prefix your pattern with "*" e.g. *lisp to list all lisp files, you
can separate with a "," e.g. *sh,py will list only sh and py files.
OTOH you can type *!lisp to list all buffers but lisp ones, etc...

> This is a specific example of a more generic question of how to search
> in other fields -- although, helm-mini might be the only one that has
> multiple fields to search in.

Try C-h m while in helm-mini to have more documentation for this.

>
> The other question is about Github... how can I get notifications of new discussions here?

Hmm, I received this message by mail, it is how I saw it, but I don't
know/remember if I had configured something on github for this, don't
think so.

> Oh... sorry, one last question: I date back to when memory was
> precious so killing buffers is a thing I do. Call it a mental
> illness. Can I kill any buffer with a name starting with *helm-.

You can but it is not a good idea as keeping those buffers allow you to
resume the corresponding helm sessions.

> I think I read somewhere that I could but just wanted to check.

If somewhere is reddit, better being careful on what you read there, at
least for helm...


-- 
Thierry

**Reply by pedz** [reply]
Note to others.  It appears that the Major mode name must be in lower case.  e.g. *Dired does not work but *dired does.  Fine... after you figure It out.  🤔

**Comment by thierryvolpiatto** [comment]
For notifications of new discussions here, it is finally not clear how it works, it seems we have to answer and then mark as answer what we already answered...???

**Comment by thierryvolpiatto** [comment]
Also it seems we can reply or answer, don't know what the difference is...

**Comment by pedz** [comment]
I got this reply via email but I am pretty sure I don’t get any notification of new threads.  Watching others questions / answers is a good way to learn.

Thank you for all your help.  I did do C-h m … oh wait… I did C-h b to get the bindings.  Ok… Thanks again.

**Comment by thierryvolpiatto** [comment]
Perry Smith ***@***.***> writes:

> I got this reply via email but I am pretty sure I don’t get any
> notification of new threads. Watching others questions / answers is a
> good way to learn.

I am not sure it works well on github, I am receiving today posts from
July!

-- 
Thierry
